src/components/timeline/Markers.js

```diff
@@ -126,8 +121,9 @@ class TimelineMarkersImplementation extends React.PureComponent<Props, State> {
       if (time < start || time >= start + dur) {
         continue;
       }
-      const style = name in styles ? styles[name] : styles.default;
-      if (y >= style.top && y < style.top + style.height) {
+      const markerStyle =
```

Comment:
I found it really confusing being called `styles`, as I assumed it was CSS styles. I made it a bit more precise with `markerStyles`.